azurerm/resource_arm_cosmosdb_account.go

```diff
@@ -34,6 +34,13 @@ func resourceArmCosmosDbAccount() *schema.Resource {
 			State: schema.ImportStatePassthrough,
 		},
 
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(120 * time.Minute),
+			Read:   schema.DefaultTimeout(5 * time.Minute),
+			Update: schema.DefaultTimeout(120 * time.Minute),
+			Delete: schema.DefaultTimeout(120 * time.Minute),
```

Comment:
the deployment time for CosmosDB depends on how many replica's are specified - as such I think it'd be prudent to set these to `180m`